explainaboard/explainaboard_main.py

```diff
@@ -471,11 +472,12 @@ def load_system_details_path():
             # metadata[
             #     "user_defined_features_configs"
             # ] = loader.user_defined_features_configs
-            metadata["task_name"] = task
+            metadata_one = copy.deepcopy(metadata)
```

Comment:
>"It looks more informative name e.g., metadata_copied is better."
Good suggestion.

>"It is also unclear why we need to perform deepcopying here."

It will result in a bug without deepcopy when performing multiple system analyses due to this: https://github.com/neulab/ExplainaBoard/blob/fa21e64e54341cd1554d27ba78fa139eeed1fc30/explainaboard/analysis/analyses.py#L35

Reply:
Thanks, it looks the cause comes from mutable access to a dict that is not a topic of this PR. Please skip the latter part of the comment.